CHANGELOG.md

```diff
@@ -13,9 +13,14 @@ Remember to align the itemized text with the first line of an item within a list
     bumped to 0.4.0 but this has been rolled back in this release to give users
     of both TensorFlow and JAX more time to migrate to a newer TensorFlow
     release.
+  * jax now depends on jaxlib directly. This change was enabled by the CUDA
```

Comment:
Maybe add, so now you can just do `pip install jax` instead of `pip install jax[cpu]`?